services/userService.js

```diff
@@ -0,0 +1,39 @@
+const { set, get } = require('./cacheService')
+const {
+  fetchUser
+} = require('../models/users')
+
+const cacheUser = (user) => {
+  if (!user) {
+    return false
+  }
+  const userObject = {
+    github_id: user.github_id,
+    id: user.id,
+    username: user.username
+  }
+  set(user.id, userObject)
```

Comment:
What about the TTL?
Do we need to store this forever? We will need to modify this if the username is changed.